config.yaml

```
@@ -1,3 +1,4 @@
 mods:
     - name: 'AutoDeconstruct'
-    - name: 'Squeak Through'
\ No newline at end of file
+    - name: 'Squeak Through'
+    - name: 'Todo List'

```

Should be 'Todo-List' based on URL info
```suggestion
    - name: 'Todo-List'
```